suite-native/module-device-settings/src/screens/FirmwareUpdateScreen/FirmwareChangelog.tsx

```diff
@@ -0,0 +1,80 @@
+import { useMemo } from 'react';
+import { useSelector } from 'react-redux';
+
+import { BottomSheet, Button, Text } from '@suite-native/atoms';
+import { selectFirmwareChangelog } from '@suite-common/wallet-core';
+import { Translation } from '@suite-native/intl';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+type FirmwareChangelogProps = {
+    isVisible: boolean;
+    onClose: () => void;
+};
+
+const changelogSectionTitleTextStyle = prepareNativeStyle(utils => ({
+    ...utils.typography.highlight,
+    paddingTop: utils.spacings.sp24,
+}));
+
+const buttonContainerStyle = prepareNativeStyle(utils => ({
+    marginTop: utils.spacings.sp32,
+}));
+
+const ChangelogSectionTitle = ({ children }: { children: React.ReactNode }) => {
+    const { applyStyle } = useNativeStyles();
+
+    return <Text style={applyStyle(changelogSectionTitleTextStyle)}>{children}</Text>;
+};
+
+export const FirmwareChangelog = ({ isVisible, onClose }: FirmwareChangelogProps) => {
+    const firmwareChangelog = useSelector(selectFirmwareChangelog);
+    const { applyStyle } = useNativeStyles();
+
+    const formattedChangelog = useMemo(() => {
+        if (!firmwareChangelog) {
+            return (
+                <Text>
+                    <Translation id="moduleDeviceSettings.firmware.firmwareUpdateScreen.changelog.changelogUnavailable" />
+                </Text>
+            );
+        }
+
+        if (typeof firmwareChangelog === 'string') {
+            return <Text>{firmwareChangelog}</Text>;
+        }
+
+        return firmwareChangelog.map((text, index) => {
+            const key = text + index;
+
+            if (text.startsWith('#')) {
+                const strippedText = text.replaceAll('#', '').trim();
+
+                return <ChangelogSectionTitle key={key}>{strippedText}</ChangelogSectionTitle>;
+            }
+
+            let formattedText = text;
+
+            if (text.startsWith(' - ')) {
```

Comment:
I guess you might want to do the same for '*' since it looks like this otherwise:
![image](https://github.com/user-attachments/assets/bb996d9c-1d6e-43fc-8f49-2a88e3f40b88)